example/android/app/src/main/AndroidManifest.xml

```diff
@@ -6,7 +6,7 @@
          additional functionality it is fine to subclass or reimplement
          FlutterApplication and put your custom class here. -->
     <application
-        android:name="io.flutter.app.FlutterApplication"
+        android:name="${applicationName}"
```

Comment:
See https://github.com/flutter/flutter/wiki/Upgrading-pre-1.12-Android-projects for explanation of changes.